### PR TITLE
Add sidebar links

### PR DESCRIPTION
**Pull request by hbillings**
Fixes issue(s) #2825  .

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/2825-add-sidebar-links)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/2825-add-sidebar-links/blog/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/2825-add-sidebar-links/README.md)

Changes proposed in this pull request:
-
-
-

Checklist:
- [ ] All images being added have been optimized **_--> [learn more](18f.gsa.gov/styleguide/images/#using-and-optimizing-jpg-and-png) about how to optimize images <--_**

/cc @relevant-people

### CODE REVIEW

File: _sass/_components/blog-index.scss [hunk 1]
```diff
@@ -30,6 +41,22 @@
...
+  background-color: $color-gray-lightest;
+  padding: 20px;
+
+  ul {
+    margin-bottom: 0;
+  }
+
+  li {
+    margin-bottom: $paragraph-margins-thick;
+    &:last-child {
```
[the remark this answers is not in the document]
Comment (maya) [reply _sass/_components/blog-index.scss]: Fixed in 9652437.

File: _sass/_components/blog-index.scss [hunk 2]
```diff
@@ -1,25 +1,53 @@
...
-        display: initial;
-      }
-    }
+.blog-footer {
+  h2 {
+    font-size: $base-font-size;
+    font-weight: normal;
+  }
+
+  a.usa-button {
```
[the remark this answers is not in the document]
Comment (maya) [reply _sass/_components/blog-index.scss]: Fixed in 9652437.

**Comment by maya** [comment]
This is ready for review @eddietejeda.

**Comment by awfrancisco** [comment]
Looks good to me!

**Pull request merged**